rules/common.smk

```diff
@@ -12,6 +12,18 @@ def subtypes_by_subtype_wildcard(wildcards):
     }
     return(db[wildcards.subtype])
 
+
+def no_sign_request(_wildcards):
```

Comment:
Would a better scope for this function be under the `else` next to `rule download_metadata` instead of at the top level?

Reply:
This no longer applies since the refactoring in https://github.com/nextstrain/avian-flu/pull/72 removed the if/else here.